R/pdf_document.R

```diff
@@ -170,8 +170,8 @@ pdf_document <- function(toc = FALSE,
 
     input_test <- read_utf8(input_file)
 
-    # set the margin to 1 inch if no other geometry options specified
-    if (!has_yaml_parameter(input_test, "geometry"))
+    # set the margin to 1 inch if no geometry options or document class specified
+    if (!has_yaml_parameter(input_test, "geometry") || !has_yaml_parameter(input_test, "documentclass"))
```

Comment:
You can actually use a regular expression here:
```suggestion
    if (!has_yaml_parameter(input_test, "(geometry|documentclass)"))
```

Reply:
I did not know that; thank you!